### PR TITLE
Added dump_serializers command for debugging

### PR DESCRIPTION
**Pull request by vdboor**
Just like `manage.py dump_models` we now have `manage.py dump_serializers`.

### CODE REVIEW

File: src/dso_api/dynamic_api/management/commands/dump_serializers.py [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+                doc += "\n"
+        self.stdout.write(f'    """{doc}"""\n\n')
+
+    def write_factory_docs(self, serializer_class: Type[DynamicSerializer]) -> None:
+        """Write debugging information about the factory function."""
+        # Write some factory information
+        factory_name = self._get_serializer_factory_name(serializer_class)
+        if issubclass(serializer_class, DynamicSerializer):
+            self.stdout.write(f"    # Set by {factory_name}():\n")
+            self.stdout.write(f"    # table_schema = {serializer_class.table_schema}\n")
```
Comment (gkoller) [inline src/dso_api/dynamic_api/management/commands/dump_serializers.py:169]: Minor nitpicking. You seem to be doing identation manually. That soon starts to become errorprone, do I have 7 or 8 spaces (=level 2), 11 or 12 (=level 3), etc. You could do something like this:

```python
ident = lambda level: " " * 4 * level
...
self.stdout.write(f"{ident(1)}# Set by {factory_name}():\n")
self.stdout.write(f"{ident(1)}# table_schema = {serializer_class.table_schema}\n")
...
... 
self.stdout.write(f"{ident(2)}{name} = {value!r}\n")
```
Looks weird, but it is immediately clear what indentation level you are on.

**Review by gkoller — APPROVED** [review]

File: src/dso_api/dynamic_api/management/commands/dump_serializers.py (hunk 1 above)
Comment (lbam) [reply src/dso_api/dynamic_api/management/commands/dump_serializers.py:169]: Less verbose:

```python
def indented(level, s):
    self.stdout.write(" " * 4 * level)
    self.stdout.write(s)

Comment (gkoller) [reply src/dso_api/dynamic_api/management/commands/dump_serializers.py:169]: @lbam That's a much better solution. Thanks!

Comment (vdboor) [reply src/dso_api/dynamic_api/management/commands/dump_serializers.py:169]: well in this case (and with the dump_models) command, there is only one indentation level. This won't change either so I'd vote for keeping the code as it is

**Pull request merged**